packages/core/src/accessibility/focusStyleManager.ts

```diff
@@ -4,12 +4,20 @@
  * of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
  * and https://github.com/palantir/blueprint/blob/master/PATENTS
  */
+/* istanbul ignore next */
```

Comment:
Seems to just ignore the next entire statement block. https://github.com/gotwarlost/istanbul/blob/master/ignoring-code-for-coverage.md